plugins/user-settings-backend/src/service/router.ts

```diff
@@ -104,6 +108,14 @@ export async function createRouterInternal(options: {
       key,
     });
 
+    if (options.signals) {
+      await options.signals.publish<{ key: string }>({
+        recipients: { type: 'user', entityRef: userEntityRef },
+        channel: `user-settings`,
+        message: { key },
```

Comment:
I don't know the common granularity people are meant to use here, but i'd expected some form of typing. Either

```
channel: 'user-settings:key-changed'
```

or

```
message: { type: 'setting-changed', key }
```